core/src/test/java/org/opentosca/toscana/core/api/TransformationControllerTest.java

```diff
@@ -69,6 +71,42 @@ public void setUp() throws Exception {
         mvc = MockMvcBuilders.standaloneSetup(controller).build();
     }
 
+    //<editor-fold desc="Start transformation tests">
+
+    @Test
+    public void testStartTransformationSuccess() throws Exception {
+        preInitNonCreationTests();
+        transformationService.setStartReturnValue(true);
+        mvc.perform(
+            get("/csars/k8s-cluster/transformations/p-a/start")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(VALID_PROPERTY_INPUT)
+        ).andDo(print())
+            .andExpect(status().is(200))
+            .andExpect(content().bytes(new byte[0]))
+            .andReturn();
+        assertEquals(TRANSFORMING,
+            csarService.getCsar("k8s-cluster").getTransformations().get("p-a").getState());
```

Comment:
Please introduce fields for `platformName` and `sarName`, hardcoded Strings are hard to maintain.